README.md

```diff
@@ -49,6 +49,12 @@ cf set-env $YOUR_APP_NAME DD_LOGS_CONFIG_TCP_FORWARD_PORT 10514
 cf set-env $YOUR_APP_NAME DISABLE_STD_LOG_COLLECTION true
 ```
 
+To collect metrics with DogStatsD only, it is possible to disable metric collection in the puppy agent by adding:
+
+```
+cf set-env $YOUR_APP_NAME DD_ENABLE_CHECKS false
```

Comment:
I dont believe we support checks here at the moment. StatsD only.

Reply:
Apparently core checks are run by default on the agent and that would generate metrics and logs (as the agent logs when trying to run a check).

@arbll for confirmation

Reply:
The puppy runs some "core checks" by default : cpu/ntp/...

Reply:
They are not regular python checks but things that are built in the agent/puppy

Reply:
@arbll do we have a list of those?

Reply:
not sure if we have them listed somewhere 

they live in the agent repo :
https://github.com/DataDog/datadog-agent/tree/master/pkg/collector/corechecks

I can find
https://docs.datadoghq.com/integrations/system/
https://docs.datadoghq.com/integrations/ntp/

The snmp one has a python equivalent and is not used by anyone yet AFAIK